images/plbase/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM amazonlinux:2023
 ARG CACHEBUST=2025-06-15-14-13-20
+# This ensures you aren't accidentally using a non-venv python in your terminal.
+ENV PATH="/PrairieLearn/.venv/bin:$PATH"
```

Comment:
The issue here is the distinction between plbase (which doesn't have /PrairieLearn) and the prairielearn image itself. I'm not sure why the distinction continues (I believe there were other images in the past that used plbase as a starting point), but one alternative to avoid changing plbase is to change the main PL Dockerfile to install the venv but reuse the system packages (`python -m venv --system-site-packages /PrairieLearn/.venv`). This would allow the docker context to still use .venv, but have the packages that are installed in plbase.

Reply:
Just to clarify, we would install to system packages in `plbase` and pull in those packages into the venv when it gets set up in `prairielearn`.

Reply:
Yes, that's what I thought. Note that (if I'm understanding the docs correctly) `--system-site-packages` allows the venv to use the system packages in addition to any venv-specific packages, so you wouldn't need to pull them manually when the venv is created.

Reply:
I'm not opposed, but I also wonder if we can just unify `plbase` and `prairielearn` now that container builds have caching.

Reply:
Done in a separate PR.